### DISCUSSION TITLE
Failed to start velero in a kubernetes, although it should be done easily and quickly

### DISCUSSION THREAD

**Discussion by xueliangyang-oeuler** [vmware-tanzu/velero#5952] [Community support Q&A]
Though a little long time, I failed to make velero run successfully under a kubernetes circumstance, and I tried to use aliyun oss as an object storage. I need some help for this question.

**Comment by anshulahuja98** [comment]
Please try to provide more details
Such as the 
1. steps done to install velero
2. pod logs  of velero 
3.  the BackupStorageLocation values 
4. status of BackupStorageLocation  with any error messages.
